odoo/addons/base/models/res_lang.py

```diff
@@ -334,6 +334,23 @@ def format(self, percent, value, grouping=False, monetary=False):
 
         return formatted
 
+    def action_activate_langs(self):
+        """ Activate the selected languages """
+        for lang in self:
+            lang.active = True
```

Comment:
You should probably call ``toggle_active`` which changes the active status and does model-specific behavior on active flag change (here updating modules translations).